react_demo/index.html

```diff
@@ -7,8 +7,8 @@
   </head>
   <body>
     <div id="c5t-root">Loading...</div>
-    <script src="./vendor/react-0.13.3-build/react.js"></script>
-    <script src="./vendor/react-0.13.3-build/JSXTransformer.js"></script>
+    <script src="./vendor/react/react.js"></script>
```

Comment:
(From the phone)

It's the other index.html you're commenting on, the one outside the C++
directory, right?

On Saturday, June 6, 2015, John Babak <notifications@github.com> wrote:

> In react_demo/index.html
> <https://github.com/KnowSheet/SimpleDashboard/pull/7#discussion_r31874017>
> :
>
> > @@ -7,8 +7,8 @@
> >    </head>
> >    <body>
> >      <div id="c5t-root">Loading...</div>
> > -    <script src="./vendor/react-0.13.3-build/react.js"></script>
> > -    <script src="./vendor/react-0.13.3-build/JSXTransformer.js"></script>
> > +    <script src="./vendor/react/react.js"></script>
>
> Does not match the path you serve.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/KnowSheet/SimpleDashboard/pull/7/files#r31874017>.
>


-- 
And, as always, have a nice day!
Dima

http://dimakorolev.com/ | http://www.linkedin.com/in/dimakorolev |
dmitry.korolev@gmail.com | +1 (312) 593 2783